source/index.html.md

```diff
@@ -25,6 +25,7 @@ includes:
   - 150-dashboard
   - 160-pipeline-config
   - 170-version
+  - 171-pluggable-scms
```

Comment:
The file is called `_170-pluggable-scms.md.erb`, so your documentation is not being rendered in the main page. I'm fairly certain.

Also wonder why middleman does not raise a ruckus about missing partials.